src/Language/PureScript/Interactive.hs

```diff
@@ -190,9 +190,8 @@ handleShowImportedModules
   => (String -> m ())
   -> m ()
 handleShowImportedModules print' = do
-  PSCiState { psciImportedModules = importedModules } <- get
-  print' $ showModules importedModules
-  return ()
+  st <- get
+  print' $ showModules (psciImportedModules st)
```

Comment:
Bit of a nitpick, but using `$` as well as parens on the same line is stylistically perhaps not the best thing to do. I don't mind which but would you mind sticking to one or the other?